packages/example-generator/src/run.ts

```diff
@@ -185,7 +188,12 @@ async function checkExamples(
   resolvedOutput: string,
   tmpDir: tmp.DirResult,
 ): Promise<void> {
-  for (const [template, sdkVersion] of templatesWithSdkVersions(TEMPLATES)) {
+  for (
+    const [template, sdkVersion] of [
+      ...templatesWithSdkVersions(TEMPLATES),
+      ...templatesWithSdkVersions(WIDGET_TEMPLATES),
```

Comment:
There were a bunch of places here not applying properly to the widget templates so weren't actually beingg enforced at CI time